src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -9336,5 +9336,12 @@ internal MyException(int error, int offset, string message) : base(message)
     string.Format(FeaturesResources.Generate_parameter_0, "Error", "MyException"),
 });
         }
+
+        [WorkItem(48172, "https://github.com/dotnet/roslyn/issues/48172")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateVariable)]
+        public async Task TestMissingOfferParameterInTopLevel()
+        {
+            await TestMissingAsync("[|Console|].WriteLine();", new TestParameters(Options.Regular));
```

Comment:
This test is for Regular C# only.
C#-script have a different behavior which is already tested in other tests.